packages/dashboard/src/pages/Wallets/components/CollectibleList/index.tsx

```diff
@@ -88,15 +87,10 @@ export const CollectibleList = memo<CollectibleListProps>(({ selectedNetwork })
     )
 
     useEffect(() => {
-        PluginMessages.Wallet.events.erc721TokensUpdated.on(() => retry())
-        PluginMessages.Wallet.events.socketMessageUpdated.on((info) => {
```

Comment:
#why_remove_getNonFungibleAssets

This is weird, to get the data we need to call an extra retry().